### PR TITLE
Add value chip drop-down with default aggregate functions.

### PR DESCRIPTION
**Pull request by MayaKirova**
Closes #  

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Comment by mddragnev** [comment]
@MayaKirova 
Having this configuration: 
```
values: [
            {
                member: 'NumberOfUnits',
                aggregate: {
                    aggregator:  IgxNumberSummaryOperand.sum,
                    key: 'sum',
                    label: 'Sum'
                },
                enabled: true

            }, {
                member: 'Value',
                aggregate: {
                    aggregator:  IgxNumberSummaryOperand.sum,
                    key: 'sum',
                    label: 'Sum'
                },
                enabled: true,
                formatter: (val) => val ? parseFloat(val).toFixed(2) : undefined
            }
        ],
```
or you can observe the following behavior in hierarchy sample.
On first interaction with the value chip dropdown there are two possible aggregators - the one specified in the configuration and the default one for the numeric type. When the default one is selected and when you reopen the dropdown, there is only the default aggregator. IMO we should always show the configuration one.

**Comment by dkamburov** [comment]
@MayaKirova please resolve conflicts

File: projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-aggregate.ts [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+     * ```typescript
+     * IgxPivotAggregate.aggregators();
+     * ```
+     *
+     * @memberof IgxPivotAggregate
+     */
+    public static aggregators() {
+        return [{
+            key: 'COUNT',
+            label: 'Count',
```
Comment (dkamburov) [inline projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-aggregate.ts:17]: All labels should be localized

File: projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.pipes.spec.ts [hunk 2]
```diff
@@ -61,7 +62,11 @@ describe('Pivot pipes', () => {
             values: [
                 {
                     member: 'UnitsSold',
-                    aggregate: IgxNumberSummaryOperand.sum,
+                    aggregate: {
+                        aggregator: IgxNumberSummaryOperand.sum,
```
Comment (dkamburov) [inline projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.pipes.spec.ts:66]: Looks like it's missed to change to `IgxPivotNumericAggregate`

File: src/app/pivot-grid-hierarchy/pivot-grid-hierarchy.sample.ts [hunk 3]
```diff
@@ -47,12 +47,20 @@ export class PivotGridHierarchySampleComponent {
         values: [
             {
                 member: 'NumberOfUnits',
-                aggregate: IgxNumberSummaryOperand.sum,
+                aggregate: {
+                    aggregator:  IgxNumberSummaryOperand.sum,
```
Comment (dkamburov) [inline src/app/pivot-grid-hierarchy/pivot-grid-hierarchy.sample.ts:51]: also should be `IgxPivotNumericAggregate`

File: src/app/pivot-grid-hierarchy/pivot-grid-hierarchy.sample.ts [hunk 4]
```diff
@@ -47,12 +47,20 @@ export class PivotGridHierarchySampleComponent {
...
+                    key: 'sum',
+                    label: 'Sum'
+                },
                 enabled: true
 
             }, {
                 member: 'Value',
-                aggregate: IgxNumberSummaryOperand.sum,
+                aggregate: {
+                    aggregator:  IgxNumberSummaryOperand.sum,
```
Comment (dkamburov) [inline src/app/pivot-grid-hierarchy/pivot-grid-hierarchy.sample.ts:60]: also should be `IgxPivotNumericAggregate`

File: src/app/pivot-grid-noop/pivot-grid-noop.sample.ts [hunk 5]
```diff
@@ -54,7 +54,11 @@ export class PivotGridNoopSampleComponent {
         values: [
             {
                 member: 'UnitsSold',
-                aggregate: IgxNumberSummaryOperand.sum,
+                aggregate: {
+                    aggregator:  IgxNumberSummaryOperand.sum,
```
Comment (dkamburov) [inline src/app/pivot-grid-noop/pivot-grid-noop.sample.ts:58]: also should be IgxPivotNumericAggregate

**Review by dkamburov — APPROVED** [review]

**Pull request merged**